pkg/image/prune/imagepruner.go

```diff
@@ -94,6 +96,9 @@ type PrunerOptions struct {
 	// KeepTagRevisions is the minimum number of tag revisions to preserve;
 	// revisions older than this value are candidates for pruning.
 	KeepTagRevisions int
+	// KeepOverSizeLimitImages indicates that images exceeding defined limits (openshift.io/Image)
+	// will not be considerate as candidates for pruning.
```

Comment:
Not quite, the strongest is (or was) `keep-younger-than` which removed images younger than x. Generally we're talking here about logical sum of all passed flags. Each one can be modified by adding additional one. Thus the word keep can be seen wrong in all cases, when connected with other flags.